README.md

```diff
@@ -40,7 +40,8 @@ The `pkg-fun.nix` files in all unit directories are automatically discovered, ca
 
 These requirements will be checked using CI:
 1. The `pkgs/unit` directory must only contain unit directories, and only in subdirectories of the form `${substring 0 4 name}/${name}`.
-2. <a id="req-ref"/> Files outside a unit directory must not reference files inside that unit directory, and the other way around.
+2. <a id="req-ref-out"/> Files inside a unit directory must not reference files outside that unit directory.
+3. <a id="req-ref-in"/> Files outside a unit directory must not reference files inside a unit directory, except for definitions of variant attributes¹ in `all-packages.nix` and the auto-calling logic.
```

Comment:
```suggestion
3. <a id="req-ref-in"/> Files outside a unit directory must not reference files inside a unit directory, except for definitions of variant attributes¹ in `all-packages.nix`.
```

Mentioning the auto-calling logic here only serves to make the proposal _technically_ correct, but distracts from the purpose of the line.

Reply:
How about this: https://github.com/nixpkgs-architecture/simple-package-paths/pull/27/files#r1082872667